Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -281,7 +284,7 @@ extension File {
             return (false, end - start)
         }
 
-        let count = end - start - numberOfCommentAndWhitespaceOnlyLines(startLine: start, endLine: end)
+        let count = end - start - numberOfCommentAndWhitespaceOnlyLines(startLine: start, endLine: end) - 1
```

Comment:
Next step is to make

`let count = (start...end).count - numberOfCommentAndWhitespaceOnlyLines(startLine: start, endLine: end)` to include startLine and endLine and update the `function_body_length` and `type_body_length` to pass new startLine and endLine values.